ckanext/datagovuk/action/create.py

```diff
@@ -67,7 +69,11 @@ def resource_create(context, data_dict):
 
             if errors:
                 context['session'].rollback()
-                raise ValidationError(errors)
+                error_summary = OrderedDict()
```

Comment:
This could be written as:

```python
error_items = [(str(i + 1), error) for i, error in enumerated(error)]
error_summary = OrderedDict(error_items)
```

which might be more succinct.

Reply:
sure